SensorFusion_src/contribution.c

```diff
@@ -1,16 +1,30 @@
-double* contribution(double lambda[], int n)
+/***************************************************************************
+* Author : Shyam Bhuptani 
+* Function : Contribution 
+* Application : This function takes the eigenvalues and returns the alpha
+* Input : size of eigenvalues n, eigenvalue array (lambda) of size n
+* Output : alpha is an array of size n
+*/
+
+/*Including dependancies*/
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <math.h>
+#include "../include/contribution.h" 
+
+double * contribution(double lambda[], int n)
```

Comment:
You should make your function name more explicit. Ideally, the function name should describe what the function does, eg. compute_contributions(). It is unclear how the function relates to contributions, in this case.

Reply:
Updated in new commit, Please check